src/legacy/core_plugins/data/public/filter/filter_bar/filter_editor/index.tsx

```diff
@@ -265,6 +266,7 @@ class FilterEditorUI extends Component<Props, State> {
           onChange={this.onFieldChange}
           singleSelection={{ asPlainText: true }}
           isClearable={false}
+          className="filter-editor__field"
```

Comment:
Can you change the class to adhere to the BEM naming convention and apply the correct prefix of `global`? Something like
```suggestion
          className="globalFilterEditor__fieldInput"
```

Reply:
Changed the `className` as well as the class name in the SCSS file